deploy/chart/values.yaml

```diff
@@ -51,3 +51,5 @@ package:
     requests:
      cpu: 10m
      memory: 50Mi
+  securityContext:
```

Comment:
Can you move this to `deploy/upstream/values.yaml`? Then we should be good to merge. Thanks!

Reply:
Done!